pkg/api/dashboard.go

```diff
@@ -81,8 +85,9 @@ func dashboardGuardianResponse(err error) response.Response {
 // 404: notFoundError
 // 500: internalServerError
 func (hs *HTTPServer) GetDashboard(c *contextmodel.ReqContext) response.Response {
-	ctx, span := hs.tracer.Start(c.Req.Context(), "api.GetDashboard")
+	ctx, span := tracer.Start(c.Req.Context(), "api.isDashboardStarredByUser")
 	defer span.End()
+	c.Req = c.Req.WithContext(ctx)
```

Comment:
Kristin's comment was resolved but I still see it - the span name here is inconsistent with the method name

Reply:
I forgot to push! Thank you.